examples/tpm_test.h

```diff
@@ -506,6 +506,150 @@ static const byte kTestAesCfb128Msg[] = {
     0xe5,0xfb,0xc1,0x19,0x1a,0x0a,0x52,0xef
 };
 
+#include <wolftpm/tpm2_wrap.h>
+
+inline static int getPrimaryStoragekey(WOLFTPM2_DEV* pDev,
```

Comment:
Traditionally its `static inline`, but either probably works.
I'd be fine with creating a `examples/tpm_test.c` and putting these function in a .c with better naming...